### ISSUE TITLE
Remove deck.gl version pinning

### ISSUE THREAD

**Issue by chrisgervang** [visgl/hubble.gl#251]
Root cause: the kepler.gl example needs deck.gl 8.2 to match the version used in kepler.gl. Currently this example is built into the hubble.gl website, so that is required to also pin to deck.gl 8.2. This pinning is cascaded throughout the repo currently, but if the kepler example is built separately from the rest of the website and examples we should be able to remove that pinning and always use the latest deck.gl version (or any version a user prefers to use).

- [x] build keplerlgl example eperatly from other website examples (#255, #295)
  - [x] deck.gl does this for the playground example. hubble needs to refactor the website build scripts to use deck's newer technique that allows for seperation.
- [ ] remove version pins throughout repo.
  - [ ] remove resolutions
  - [x] use peer dependencies where required based on minimum major API version compatibility
  - [x] pin the kepler example, as needed.
- [x] build and test website
  - [x] deploy and test
- [x] release hubble.gl patch that removes pinning

**Comment by charlieforward9** [comment]
Seems like a close duplicate of #277 , #283 ? This would be very helpful.

**Comment by chrisgervang** [comment]
Kepler has been built separately as of #255, and this is maintained in the website re-write in #295.

**Comment by charlieforward9** [comment]
Sweet! This can be closed?

**Comment by chrisgervang** [comment]
Yup! Deck is listed as a peer dependency now, and the kepler components remain injected - all other deps shouldn't need to be peered
